src/twig/_patterns/02-molecules/menus/breadcrumbs/breadcrumbs.twig

```diff
@@ -15,7 +15,7 @@
                 icon_name: 'arrow-xsmall',
                 icon_blockname: breadcrumbs__base_class,
                 icon_modifiers: ['small'],
-              } %}
+              } %}            
```

Comment:
Seems like you might have just added some whitespace here? Are you showing whitespace in your editor? It will help with committing sort of random gunk like this

Reply:
Ugh I had a bunch of format-on-save stuff from before, which didn't play well with my editor not being able to understand `.twig` files